### PR TITLE
Show receipts in tables, fixes #192

### LINKED ISSUE #192: Refactor statements as tables
The ``format*Statements`` functions from [ReceiptService](https://github.com/nemtech/nem2-cli/blob/master/src/services/receipt.service.ts ) formats statements not using tables and the output is not tested enough.

To solve these issues, one option is to:

1) Define a table per statement  (TransactionStatement, MosaicResolutionStatement, AddressResolutionStatement) under the folder ``views/receipt``.
2) Add unit tests to each table.
3) Replace ``nem2-cli block receipts`` ReceiptService dependency and display the receipts using the new tables instead.

A similar solution has been applied for transactions in #199 . To display inner receipts, please review how inner aggregate transactions are being rendered inside the same table.

### PR DESCRIPTION
**Pull request by decentraliser**

### CODE REVIEW

File: src/views/statements/transactionStatements.view.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+ private renderReceipt(
+  receipt: Receipt,
+  index: number,
+  numberOfReceipts: number,
+ ): CellRecord[] {
+  const {type, size} = receipt
+  return [
+   this.getSectionTitle(`Receipt ${index + 1} of ${numberOfReceipts}`),
+   {type: ReceiptType[type]},
+   {size: size || 'N/A'},
```
Comment (dgarcia360) [inline src/views/statements/transactionStatements.view.ts]: Size seems to be always N/A 
image

File: src/views/statements/transactionStatements.view.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  */
+ private renderReceipt(
+  receipt: Receipt,
+  index: number,
+  numberOfReceipts: number,
+ ): CellRecord[] {
+  const {type, size} = receipt
+  return [
+   this.getSectionTitle(`Receipt ${index + 1} of ${numberOfReceipts}`),
+   {type: ReceiptType[type]},
```
Comment (dgarcia360) [inline src/views/statements/transactionStatements.view.ts]: ``size`` and ``type`` first letters should be capitalized  to match other properties
image

File: src/views/statements/transactionStatements.view.ts (hunk 1 above)
Comment (decentraliser) [reply src/views/statements/transactionStatements.view.ts]: This looks ok to me, except if 0 is a valid size?

**Comment by decentraliser** [comment]
Below is what I came up to with styling
I think it is pretty readable like that

image

**Pull request merged**